packages/turbo-codemod/src/transforms/migrate-env-var-dependencies.ts

```diff
@@ -93,11 +91,11 @@ export function migrateGlobal(config: TurboJsonSchema) {
 
 export function migrateConfig(config: TurboJsonSchema) {
   const migratedConfig = migrateGlobal(config);
-  Object.keys(config.pipeline).forEach((pipelineKey) => {
-    config.pipeline;
-    if (pipelineKey in config.pipeline) {
-      const pipeline = migratedConfig.pipeline[pipelineKey];
-      migratedConfig.pipeline[pipelineKey] = {
+  Object.keys(config.tasks).forEach((pipelineKey) => {
+    config.tasks;
```

Comment:
I'm leaving this in because it's out of scope for the PR, but as far as I can tell there's no need for this statement